.ruby-version

```diff
@@ -1 +1 @@
-ruby-2.3.1
+ruby-2.4.3
```

Comment:
Is this the version on the server? It might be nice to have this in a separate commit.

Reply:
Sure, split out to a new commit.

Reply:
Thank you! And thank you for fixing it.